docs/references/context/page.md

```diff
@@ -28,30 +28,45 @@ parts of the request.
   // Consider the path to be /employee/{id}
   id := ctx.Request.PathParam("id")
   ```
-- `Bind(interface{})` - to access a decoded format of the request body, the body is mapped to the interface provided 
-  ```go
-  // incoming request body is 
-  // {
-  //    "name" : "trident",
-  //    "category" : "snacks"
-  // }
+  - `Bind(interface{})` - to access a decoded format of the request body, the body is mapped to the interface provided 
+    ```go
+    // incoming request body is 
+    // {
+    //    "name" : "trident",
+    //    "category" : "snacks"
+    // }
   
-  type product struct{
-      Name string `json:"name"`
-      Category string `json:"category"`
-  }
+    type product struct{
+        Name string `json:"name"`
+        Category string `json:"category"`
+    }
 
-  var p product 
-  ctx.Bind(&p)
-  // the Bind() method will map the incoming request to variable p
-  ```
-- `HostName()` - to access the host name for the incoming request
-  ```go
-  // for example if request is made from xyz.com
-  host := ctx.Request.HostName()
-  // the host would be http://xyz.com
-  // Note: the protocol if not provided in the headers will be set to http by default
-  ``` 
+    var p product 
+    ctx.Bind(&p)
+    // the Bind() method will map the incoming request to variable p
+    ```
+    - `HostName()` - to access the host name for the incoming request
+      ```go
+      // for example if request is made from xyz.com
+      host := ctx.Request.HostName()
+      // the host would be http://xyz.com
+      // Note: the protocol if not provided in the headers will be set to http by default
+      ``` 
+  
+    - `QueryParams()` - to access query parameters with support for multiple values
+  
+         - `Get(string)` - retrieves the first value for a given query parameter key
+             ```go
+              // Example: Request is /search?category=books&category=electronics
+              value := ctx.Request.QueryParams().Get("category")
+              // value = "books"
+            ```
+         - `GetAll(string)` - retrieves all values for a given query parameter key, including comma-separated values 
+            ```go
+            // Example: Request is /search?category=books,electronics&category=tech
+            values := ctx.Request.QueryParams().GetAll("category")
+            // values = []string{"books", "electronics", "tech"}
```

Comment:
It could be OK,but I'm surprised by the 4 (maybe 5) levels of indentation

![Screenshot_20240730_092739_GitHub.jpg](https://github.com/user-attachments/assets/6811ed3e-f196-4112-ab75-0ce751ef55f4)

![Screenshot_20240730_092755_GitHub.jpg](https://github.com/user-attachments/assets/c7404558-57ed-42a9-a806-8771fe2e5aca)

Reply:
I have aligned them for right indentation.
<img width="931" alt="Screenshot 2024-07-30 at 2 36 04 PM" src="https://github.com/user-attachments/assets/3d0355db-1666-4cdc-b7b5-01a3c6a3e340">